src/components/explore/ProportionExplorerView.svelte

```diff
@@ -95,12 +92,37 @@ h2 {
 
     <div class=body-control-set>
       <label class=body-control-set--label>Categories</label>
+      {#if bucketOptions.length > maxVisibleCategories}
+        <OptionMenu multi on:selection={(evt) => {
+          dispatch('selection', { selection: evt.detail.keys, type: 'activeBuckets' });
+        }}>
+          {#each bucketOptions as bucketOption, i (bucketOption)}
+            <Option
+              selected={activeBuckets.includes(bucketOption)}
+              key={bucketOption}
+              label={bucketOption}>
+              <!--
+                It's not currently possibly to specify a named slot for a
+                component, so we need to wrap this in a div.
+
+                https://github.com/sveltejs/svelte/issues/1037
+              -->
+              <div slot="right">
+                <ColorSwatch
+                  border="true"
+                  color={bucketColorMap(bucketOption)} />
```

Comment:
I think we should lighten up the default color swatch to be `var(--cool-gray-subtle)`. I suspect you put the black border because this blends in with the background. The functino that generates this color map could also just use a lighter cool gray instead.

Reply:
I'm not sure `var(--cool-gray-subtle)` is visible enough on a white background. I'll have to think about this.